src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionCommentFilter.java

```diff
@@ -179,7 +179,7 @@ public enum TagType {
      * are reassigned to the next FileContents, at which time filtering for
      * the current FileContents is finished.
      */
-    private WeakReference<FileContents> fileContentsReference = new WeakReference<>(null);
+    private WeakReference<FileContents> fileContentsReference;
 
```

Comment:
It will be initialized from setFileContents from the accept method
https://github.com/checkstyle/checkstyle/blob/00f1780694eefe400bd08c37d255772da82e1279/src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionCommentFilter.java#L218-L220